### PR TITLE
Fix demo programs 3 and 33

### PR DESCRIPTION
**Pull request by sandraros**
To complete #940

### CODE REVIEW

File: src/demos/zdemo_excel3.prog.abap [hunk 1]
```diff
@@ -104,47 +114,85 @@
...
+  DATA: lt_lines TYPE TABLE OF string.
+
+  APPEND 'AA;American Airlines;USD;http://www.aa.com       ' TO lt_lines.
+  APPEND 'AZ;Alitalia         ;EUR;http://www.alitalia.it  ' TO lt_lines.
+  APPEND 'DL;Delta Airlines   ;USD;http://www.delta-air.com' TO lt_lines.
+
+  PERFORM load_data USING lt_lines CHANGING ct_scarr.
+ENDFORM.
+
+FORM load_data USING it_data TYPE TABLE changing ct_data TYPE TABLE.
```
Comment (sandraros) [inline src/demos/zdemo_excel3.prog.abap]: @larshp I know subroutines are obsolete, but maybe you want to fix it in abaplint? If not, I will just commit the suggestion.

Comment (larshp) [reply src/demos/zdemo_excel3.prog.abap]: is the suggestion wrong?

Comment (sandraros) [reply src/demos/zdemo_excel3.prog.abap]: hmm, sorry, forget it :) I didn't see the typo in my code, `CHANGING` was lower case, so abaplint is right. I like typing `TABLE` in upper case, but I realize it's non-sense. Thanks for your help!

**Review by AndreaBorgia-Abo — APPROVED** [review]
Checker program now confirms libary produces files identical to stored references, on both NPL and A4H demo systems, LGTM!

**Pull request merged**